cmd/ocm-backplane/cloud/credentials.go

```diff
@@ -175,65 +110,38 @@ func runCredentials(cmd *cobra.Command, argv []string) error {
 	// ======== Call Endpoint ==================================
 	logger.Debugln("Getting Cloud Credentials")
 
-	credsResp, err := getCloudCredential(bpURL, clusterID)
+	var output string
+	isolatedBackplane, err := isIsolatedBackplaneAccess(cluster)
 	if err != nil {
-		return err
+		return fmt.Errorf("failed to determine if cluster is using isolated backplane access: %w", err)
 	}
-
-	// ======== Render cloud credentials =======================
-	switch cloudProvider {
-	case "aws":
-		cliResp := &AWSCredentialsResponse{}
-		if err := json.Unmarshal([]byte(*credsResp.JSON200.Credentials), cliResp); err != nil {
-			return fmt.Errorf("unable to unmarshal AWS credentials response from backplane %s: %w", *credsResp.JSON200.Credentials, err)
+	if isolatedBackplane {
+		targetCredentials, err := getIsolatedCredentials(clusterID)
+		if err != nil {
+			return fmt.Errorf("failed to get cloud credentials for cluster %v: %w", clusterID, err)
 		}
-		cliResp.Region = *credsResp.JSON200.Region
-		creds, err := renderCloudCredentials(credentialArgs.output, cliResp)
+		output, err = renderCloudCredentials(credentialArgs.output, &bpCredentials.AWSCredentialsResponse{
+			AccessKeyID:     targetCredentials.AccessKeyID,
+			SecretAccessKey: targetCredentials.SecretAccessKey,
+			SessionToken:    targetCredentials.SessionToken,
+			Expiration:      targetCredentials.Expires.String(),
+		})
 		if err != nil {
-			return err
+			return fmt.Errorf("failed to render credentials: %w", err)
 		}
-		fmt.Println(creds)
-		return nil
-	case "gcp":
-		cliResp := &GCPCredentialsResponse{}
-		if err := json.Unmarshal([]byte(*credsResp.JSON200.Credentials), cliResp); err != nil {
-			return fmt.Errorf("unable to unmarshal GCP credentials response from backplane %s: %w", *credsResp.JSON200.Credentials, err)
+	} else {
+		credsResp, err := getCloudCredential(bpURL, clusterID)
+		if err != nil {
+			return fmt.Errorf("failed to get cloud credentials for cluster %v: %w", clusterID, err)
 		}
-		creds, err := renderCloudCredentials(credentialArgs.output, cliResp)
+		output, err = renderCredentials(credsResp.JSON200.Credentials, cloudProvider)
 		if err != nil {
 			return err
 		}
-		fmt.Println(creds)
-		return nil
-	default:
-		return fmt.Errorf("unsupported cloud provider: %s", cloudProvider)
-	}
-}
-
-// GetAWSV2Config allows consumers to get an aws-sdk-go-v2 Config to programmatically access the AWS API
-func GetAWSV2Config(backplaneURL string, clusterID string) (aws.Config, error) {
```

Comment:
This function should not have been removed, osdctl depends on it https://github.com/openshift/osdctl/blob/04c8575e83b30d0199ff0d1cfd5b81761f870556/pkg/osdCloud/aws.go#L189